init.js

```diff
@@ -203,6 +203,7 @@ var spawnPoolWorkers = function(){
         } else if (!connection) {
             redisConfig = pcfg.redis;
             connection = redis.createClient(redisConfig.port, redisConfig.host);
+            connection.auth(redisConfig.password);
```

Comment:
Does this throw an error if there is no authentication?

It might be good to do two things:
1. Wrap it in an `if` statement so that we only auth if a password is set.
2. I believe a second argument is a callback, to pass an anonymous function that handles if it's unauthenticated. Maybe something like:

```javascript
if (redisConfig.password) {
    connection.auth(redisConfig.password, err => logger.error('Master', coin, `Error while attempting to authenticate redis: ${err}`))
}

Reply:
1. Yes, this makes sense. Trying to auth when password doesn't set in redis will NOT lead to an error, but I believe it's a good approach to have check.
2. thank you for suggestion, I'll implement this, test and update PR.

btw, what the best practice to update PR - should I close this one and create another or can I jsut update this?

thank you!

Reply:
You can just add more commits to this PR. Thank you!